README.md

```diff
@@ -28,11 +32,14 @@ Setup the application:
 
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
+If `gearbox setup-app` fails, try specifying `pip3` in the previous two steps.
 
-Finally, serve the application:
+Serve the application:
 
     $ gearbox serve --reload --debug
 
+Finaly, go to your web browser and navigate to `localhost:8080` to view the website.
+
 ### To use a virtual environment
```

Comment:
How did virtual env stuff end up in the README? If people choose to use this software, they should reference the official documentation on how to use it. Please remove; at most include a link to the documentation as a suggestion of an alternative development environment.

Reply:
This was added ages ago.